README.md

```diff
@@ -16,3 +16,25 @@ dxw staff use this repository as a forum to make and document technical decision
 1. Once consensus is reached and approvals given using the Github review system, the PR can be merged.
 1. When an RFC is accepted, ensure the dxw tech team is made aware of it via Slack.
 1. An RFC can be rejected. This can happen if a consensus isn't reached, or people agree rejecting it is the right thing to do. In this case the PR should be closed with a suitable comment.
+
+## How to use the GitHub review system
+
+Given that there are a [number of tools available to us when using Pull Requests](https://help.github.com/en/articles/about-pull-request-reviews) to facilitate this process we should be explicit about how we use each to communicate in order that it is clear how we can reach consensus.
+
+### Comment
+
+- Not indicative of preference and can be used for any questions, suggestions and typos
+
+### Approve
+
+- This decision SHOULD be accepted
+- A description is optional
+
+### Request changes
+
+- This decision SHOULD NOT be accepted unless the given consequences are acceptable
+- A description is required
```

Comment:
This implies requiring a second look from the reviewer, so they can clear their changes. Do we need to codify that (or a situation in which we are ok to override that and merge anyway, which might require admin rights on this repo).

Reply:
I agree that more detail on the part of requesting changes would be helpful, although I'm not clear on what would be a good idea! Generally the concept of a tie-breaker who is probably the CTO or other delegated role may be needed.

If someone requests changes I can see at least the following scenarios:

* a relevant amendment is made 
  - the reviewer preference to approved ✅
  - the reviewer does not agree with the amendment and the cycle restarts with a comment ✅
  - the reviewer does not update their preference perhaps due to being busy or general inactivity - can a tie-breaker dismiss or discount this if it's blocking consensus❓
* no relevant amendment is made
  - the author is busy or inactive - in which case the argument can be made that the proposal has become stale ✅
  - the author does not agree with the feedback or does not know how to proceed - in which case we could be at an impasse if this vote prevented a consensus and require a tie-breaker ❓
  - the team (CTO?) are comfortable with the consequences as outlined ❓

Reply:
Maybe for this round it's sufficient to ask the reviewer to be prepared to respond to comments or changes promptly?

I think we'll need a tie-breaker anyway, but that can separately, I think.